static/js/entry/django.js

```diff
@@ -38,4 +40,6 @@ document.addEventListener("DOMContentLoaded", function () {
   companiesLogoCarousel();
   successStoriesCarousel();
   applySorting();
+  applyTopicFilter();
+  applyLanguageFilter();
```

Comment:
We can create a single method and pass it params like filter class name and filter name and use those in the filter functions. It will remove the duplication.
```
export default function applyCatalogFilter(filterClassName, filterName) {
  $(filterClassName).on("click", function (event) {
    event.preventDefault();
    const url = new URL(window.location.href);
    const searchParams = url.searchParams;
    searchParams.set(filterName, $(event.target).attr("data-filter-value"));
    url.search = searchParams.toString();
    window.location.href = url.toString();
  });
}
```

Then we can do the below in django.js
```
applyCatalogFilter(".topic-filter-option", "topic"),
applyCatalogFilter(".language-filter-option", "language"),
```